node/src/BaseClient.ts

```diff
@@ -3511,6 +3519,93 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the members of a sorted set populated with geospatial information using {@link geoadd},
+     * which are within the borders of the area specified by a given shape.
+     *
+     * See https://valkey.io/commands/geosearch/ for more details.
+     *
+     * since - Valkey 6.2.0 and above.
+     *
+     * @param key - The key of the sorted set.
+     * @param searchFrom - The query's center point options, could be one of:
+     *
+     * - {@link CoordOrigin} to use the position of the given existing member in the sorted set.
```

Comment:
Need to swap `CoordOrigin` and `MemberOrigin`

Reply:
Sharp eyes! Fixed